assets/airports/airportLoadList.js

```diff
@@ -72,6 +72,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             wip: true
         },
         {
+            icao: 'efhk',
+            level: 'easy',
+            name: 'Helsinki Airport',
```

Comment:
I'm fine with any name. Officially they're now using Helsinki Airport but the old name included also -Vantaa (renamed a few year back). To make it even more confusing, I think that in Finnish, the official name was retained so it is still "Helsinki-Vantaan lentoasema".